helix-rest/src/main/java/org/apache/helix/rest/common/HelixRestNamespace.java

```diff
@@ -35,6 +35,7 @@
     METADATA_STORE_TYPE,
     METADATA_STORE_ADDRESS,
     IS_DEFAULT,
+    MULTI_ZK_ENABLED,
```

Comment:
Why we dont just rely on the system property but a new field?

Reply:
This is what the description says - please refer to the description carefully and let me know if it answers your question.